### ISSUE TITLE
The links to the source on the documentation pages seem to be broken

### ISSUE THREAD

**Issue by aplaice** [xmonad/xmonad-web#4]
For example, go to: 

http://xmonad.org/xmonad-docs/xmonad/XMonad-Core.html

and click on any of the "source" links.

Expected behaviour:
One is sent to the line of the source where the given item was defined.

Actual behaviour:
Sent to a github-pages "file not found" page.

Looking at the source, the `src` sub-directories are missing in [`xmonad`](https://github.com/xmonad/xmonad-web/tree/gh-pages/xmonad-docs/xmonad) and [`xmonad-contrib`](https://github.com/xmonad/xmonad-web/tree/gh-pages/xmonad-docs/xmonad-contrib) etc.

PS Thank you for maintaining the (otherwise) amazing documentation for an amazing window manager! :)

**Comment by dmwit** [comment]
We're hosting documentation on Hackage now. I've modified all links on xmonad.org to point to Hackage, and set up redirects for all the html pages that used to be hosted under xmonad.org/xmonad-docs to point at the top-level Hackage page for xmonad. The latter isn't ideal, but was very easy and slightly friendlier than just 404'ing them all. A patch that redirects them to their Hackage counterparts instead is very welcome. These changes should go live shortly -- I think a cron job runs daily to synch with this repo.

Along the way I deleted the gen-docs.sh script as well, since this is now an automatic part of the Hackage upload (I believe).
